cmd/libvulnhttp/main.go

```diff
@@ -13,6 +13,7 @@ import (
 	"github.com/rs/zerolog/log"
 
 	"github.com/quay/claircore/libvuln"
+	"github.com/quay/claircore/updater/defaults"
```

Comment:
underscore desired before path string ?

Reply:
I have the server explicitly check for errors down below, so it needs the name.